src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
```

Comment:
The Main class should not have a static field for FigureSupplier. It's better to create an instance where it's needed in the main method.